pvlib/ivtools/sdm.py

```diff
@@ -575,6 +578,9 @@ def fit_desoto_sandia(ivcurves, specs, const=constants, maxiter=5, eps1=1.e-3):
     .. [4] PVLib MATLAB https://github.com/sandialabs/MATLAB_PV_LIB
     """
 
+    if const is None:
+        const = {'E0': 1000.0, 'T0': 25.0, 'k': 1.38066e-23, 'q': 1.60218e-19}
```

Comment:
To late now, but a bummer to see this repeated twice. I would have preferred:

```
# top of module or in utils
CONSTANTS = {'E0': 1000.0, 'T0': 25.0, 'k': 1.38066e-23, 'q': 1.60218e-19}
```

Then
```
# in functions that use constant
def myfun(..., const=None, ...):
    if const is None:
        const = CONSTANTS
```

Reply:
@mikofski I considered making a similar suggestion but decided I'd rather see the repeated code than the public dictionary that might be deprecated in the next release. A private module-level dictionary might be the best option but it didn't seem like a big deal.

Reply:
Agreed

Reply:
@mikofski I agree, but thought we should just move ahead once the mutable default was removed, since the `CONSTANTS` dict is planned to be overhauled in v0.9 and hopefully also resolve the multiple names for the same reference quantity.